src/shared/libs/iota/extendedApi.js

```diff
@@ -539,7 +540,7 @@ const attachToTangleAsync = (provider, seedStore) => (
             // Batched proof-of-work only returns the attached trytes
             return constructBundleFromAttachedTrytes(sortTransactionTrytesArray(result), seedStore).then(
                 (transactionObjects) => ({
-                    transactionObjects,
+                    transactionObjects: orderBy(transactionObjects, 'currentIndex', ['desc']),
```

Comment:
Is this necessary when `isBundleTraversable` already does this?
https://github.com/iotaledger/trinity-wallet/blob/48d134fe9accac9269fd4346a1f5fd3ef7da9b95/src/shared/libs/iota/transfers.js#L1039

Reply:
Well `isBundleTraversable` does not mutate the original object, it's just a validation check.